drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -93,13 +93,21 @@ class Hmis::Hud::Enrollment < Hmis::Hud::Base
 
   scope :not_in_progress, -> { where.not(project_id: nil) }
 
-  scope :with_project_type, ->(project_types) do
-    wip_enrollments = joins(wip: :project).where(p_t[:project_type].in(project_types)).pluck(wip_t[:source_id])
-    nonwip_enrollments = joins(:project).where(p_t[:project_type].in(project_types)).pluck(:id)
+  scope :with_matching_projects, ->(query) do
```

Comment:
Nice, I like this approach. Maybe it should be called `:in_projects_where`?

Reply:
Agreed. I chose `with_projects_where`